### ISSUE TITLE
multi-instance segment

### ISSUE THREAD

**Issue by sjc6834** [ziqi-jin/finetune-anything#42]
hi, whether to support a fine-tuning of multi-instance segment

**Comment by AymanTalkhani** [comment]
@sjc6834 We've actually built out a platform that finetunes a research variant of SAM for labeling purposes at [InstaLabel](https://www.instalabel.ai/). All you need to do is upload your dataset, select and label a few samples you'd like to finetune the model on in our platform, and hit the 'Train annotator' button. You can then head over to the 'Annotate' tab, click 'Instant Annotation' and have the finetuned model label the rest of your dataset for you. We also offer crowdsourcing if you'd like to correct the AI predictions for verified ground truth labels. Should be useful for custom, multi-instance segmentation labeling use cases. Hope this helps!
